Modules/Loadable/Markups/MRML/vtkMRMLMarkupsROINode.h

```diff
@@ -58,16 +58,24 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsROINode : public vtkM
   /// Print out the node information to the output stream
   void PrintSelf(ostream& os, vtkIndent indent) override;
 
-  const char* GetIcon() override {return ":/Icons/MarkupsCurveMouseModePlace.png";}
+  const char* GetIcon() override {return ":/Icons/MarkupsROIMouseModePlace.png";}
+  const char* GetAddIcon() override {return ":/Icons/MarkupsROIMouseModePlaceAdd.png";}
```

Comment:
Why don't we need the place icon here as well? These names could be more clear:
- Icon: - that is shown in the subject hierarchy tree, etc.
- AddIcon: - icon with  +, to add a new markups node
- AddPointIcon or PlacePointIcon: - icon with arrow, to add a point

Reply:
@lassoan I can only see two different icons per markup in the `Markups/Resources/Icons` directory

Reply:
The basic icon type (without arrow or +) is in this folder: https://github.com/Slicer/Slicer/tree/master/Modules/Loadable/Markups/SubjectHierarchyPlugins/Resources/Icons

It would be better to define all 3 icon at the same place.

Reply:
Thanks, @lassoan. 

- I moved all icons to `Markups/Resources/Icons` 
- I renamed a couple to keep the naming consitency
- I added new `GenericMarkup` icons to be used by default (The M icon in the picture below)

Here is the result:
![screenshot](https://user-images.githubusercontent.com/1978682/111299937-d260d880-8650-11eb-95fd-94dd07b11221.png)